### PR TITLE
BTAT-7738 Welsh content correction for data entry screens

### PR DESCRIPTION
**Pull request by paulfirmstone**
# Pull Request Checklist
* [x] Branch is rebased with master
* [ ] Added Unit Tests for changed functionality
* [x] Ran `sbt clean compile coverage test it:test coverageReport`, all tests pass and coverage meets minimum
* [ ] Checked libraries are up to date with `sbt validate`. (If applicable) library updates are done in a separate, single commit
* [ ] Ran [change-vat-acceptance-tests](https://github.com/hmrc/change-vat-acceptance-tests) (see README of repo)
* [ ] Ran [change-vat-performance-tests](https://github.com/hmrc/change-vat-performance-tests) (see README of repo)
* [ ] (If applicable) raised app-config-* PRs
* [ ] (If applicable) ran Wave test for changes to views
* [ ] (If applicable) added Integration Tests

### CODE REVIEW

File: app/views/templates/inputs/dateEntryField.scala.html [hunk 1]
```diff
@@ -16,7 +16,7 @@
 
 @(field: play.api.data.Field, label: String, formClass: String, hasErrors: Boolean)(implicit messages: Messages)
 
-<label for="@field.name" class="form-group form-group-@formClass">
+<label for="@field.name" class="form-group form-group-@formClass" style="margin-right: 25px;">
```
Comment (lewishay) [inline app/views/templates/inputs/dateEntryField.scala.html:19]: Inline style like this is advised against and it's preferred to put style changes into a css file

File: conf/messages.cy [hunk 2]
```diff
@@ -125,7 +125,7 @@ capitalAssets.error.amount.noEntry = Nodwch gyfanswm gwerth yr asedion cyfalaf
 
 cannotDeregister.title = Ni all y busnes ganslo ei gofrestriad TAW
 cannotDeregister.text = Mae hyn oherwydd bod y busnes yn disgwyl y bydd ei drosiant trethadwy ar gyfer y 12 mis nesaf yn fwy na £{0}.
-cannotDeregister.link.text = Gallwch
+cannotDeregister.link.text = Gallwchƒ
```
Comment (lewishay) [inline conf/messages.cy:128]: Why was this changed?

File: app/views/templates/inputs/dateEntryField.scala.html (hunk 1 above)
Comment (paulfirmstone) [reply app/views/templates/inputs/dateEntryField.scala.html:19]: Added some custom css to our vat-v1.css file to cater for the space.

File: conf/messages.cy (hunk 2 above)
Comment (paulfirmstone) [reply conf/messages.cy:128]: This was a typo so good spot.
